scripts/cgp_data_loader.py

```diff
@@ -32,26 +31,18 @@ def main(argv=sys.argv[1:]):
                                help="Output actions that would otherwise be performed.")
     dry_run_group.add_argument("--no-dry-run", dest="dry_run", action="store_false",
                                help="Perform the actions.")
-    parser.add_argument("--dss-endpoint", metavar="DSS_ENDPOINT", required=False,
-                        default=DSS_ENDPOINT_DEFAULT,
+    parser.add_argument("--dss-endpoint", metavar="DSS_ENDPOINT", default=DSS_ENDPOINT_DEFAULT,
                         help="HCA Data Storage System endpoint to use")
-    parser.add_argument("--staging-bucket", metavar="STAGING_BUCKET", required=False,
-                        default=STAGING_BUCKET_DEFAULT,
+    parser.add_argument("--staging-bucket", metavar="STAGING_BUCKET", default=STAGING_BUCKET_DEFAULT,
```

Comment:
I propose making the staging bucket a required parameter also.
Bucket names are global, so using a default staging bucket name would only work in one, very specific context.